File: gateOne/DayThree/Java Code/TaskOne.java
import java.util.Scanner;
import java. util.Random; 
public class TaskOne {
 public static void main(String[] args) { 

	Random rand = new Random();
	Scanner scanner = new Scanner(System.in);
	
	int highestRange = 100;
	int lowestRange = 1;
	int firstNumber = 0;
	int secondNumber = 
	for(int count = 1;count <= 10;count++) {
	firstNumber = rand.nextInt(lowestRange,highestRange);
	secondNumber = rand.nextInt(lowestRange,highestRange);
	addition(firstNumber,secondNumber);
 	//multiplications(firstNumber,secondNumber);
	
	}

}
public static void multiplications(int input,int inputTwo) {

	int correctAnswer = 0;
	int WrongAnswer = 0;
	Scanner scanner = new Scanner(System.in);
	//for(int count = 1;count <= 5;count++) {

	int result = (inputTwo) * (input);

	System.out.print(inputTwo + "\tx\t" + input + "\t= ");
	int userAnwer = scanner.nextInt();

	if(userAnwer == result){correctAnswer++;}
		
	else{
		WrongAnswer++;
	   }
	System.out.println("Correct Answer is: " + correctAnswer);	
	System.out.println("Wrong Answer is: " + WrongAnswer);	
	//}
	
}
public static void addition(int input,int inputTwo) {

	int correctAnswer = 0;
	int WrongAnswer = 0;

	Scanner scanner = new Scanner(System.in);

	//for(int count = 1;count <= 10;count++) {

	int result = (inputTwo) + (input);

	System.out.print(inputTwo + "\t+\t" + input + "\t= ");
	int userAnwer = scanner.nextInt();

	if(userAnwer == result){correctAnswer++;}
		
	else{
		WrongAnswer++;
	 }
	System.out.println("Wrong Answer is: " + WrongAnswer);
	System.out.println("Correct Answer is: " + correctAnswer);	
	//}

	

}



} 	

File: gateOne/DayThree/Python Code/task_one.py
import random

correct_answer = 0
wrong_anwer = 0
result = 0

for number in range(1,11,1):

	firstNumber = random.randrange(1,100,)
	secondNumber = random.randrange(1,100,13)

	result = secondNumber * firstNumber
	print(f"{secondNumber}\tx\t{firstNumber}\t=",end =" ")
	user_answer = int(input())

	if(user_answer == result):
		correct_answer = correct_answer + 1
		print()
		print(f"The Answer: {result} is Correct")
		

	else:
		wrong_anwer = wrong_anwer + 1	
		print()
		print(f"The Answer: {user_answer} is Wrong")
		print(f"The Correct Answer is: {result}")
		print()

print()
print(f"Total Correct Answer is: {correct_answer}\nTotal Wrong Answer is: {wrong_anwer}")
